config.go

```diff
@@ -0,0 +1,32 @@
+package gospice
+
+import "os"
+
+type ClientConfig struct {
+	HttpUrl      string `json:"http_url,omitempty"`
+	FlightUrl    string `json:"flight_url,omitempty"`
+	FirecacheUrl string `json:"firecache_url,omitempty"`
+}
+
+var DEFAULT_CLIENT_CONFIG = ClientConfig{
```

Comment:
Does this really need to be a public object? Couldn't we just construct it directly in LoadConfig()?

Reply:
Smart, will change.